tests/unittest_nodes_lineno.py

```diff
@@ -0,0 +1,1185 @@
+import textwrap
+
+import pytest
+
+from astroid import builder, nodes
+from astroid.const import PY38_PLUS, PY39_PLUS, PY310_PLUS
+
+
+@pytest.mark.skipif(
```

Comment:
Do we want a test for < `3.8` to see if `end_lineno` and `end_col_offset` are correctly initiliazed as `None`?

Reply:
I'll add one function to just test one or two nodes. That should be enough. I wouldn't expect issues with it.